docs/user-guide/nlu/README.md

```
@@ -0,0 +1,34 @@
+---
+icon: brain-circuit
+---
+
+# NLU
+
+### What is NLU?
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)